### DISCUSSION TITLE
Identity UI

### DISCUSSION THREAD

**Discussion by tijno** [deso-protocol/dips#202] [Medium CIPs]
Accounts stored & apps authotised with identity can get out of hand

Eg:

<img width="223" alt="CleanShot 2022-01-31 at 15 43 58@2x" src="https://user-images.githubusercontent.com/69529928/151825278-86414c57-d02f-4fb4-bd12-c98beec27db9.png">

I think we should be able to open identity.deso.org in our browser and see:

* accounts added to identity
* apps that have access & access level granted
* last time each app authorised a transaction
* derived keys authorised and active

And edit things in the following ways:

* remove accounts completely
* backup / restore one or more account seeds (see #104)
* changes access level for specific apps
* remove access for specific apps
* remove authotisation for a derived key

I think the UI for this can be relatively simple

1. Drop down to select one of the logged in accounts
2. Shows primary action to Logout / remove account from identity
3. Shows list of apps authorised and access levels granted
4. For each app shows actions to a) change permissions, or b) remove app

**Comment by kennyjacobson** [comment]
I especially like this one: "changes access level for specific apps". 

Some DeSo-enabled apps are asking for Level 4 when they only need Level 3. I only have the choice of accepting of rejecting their initial ask for Level 4 access. If I could choose on my own to knock their Level 4 down to a Level 3 I'd feel much more comfortable with unfamiliar DeSo apps. Right now we are a small community and we know and trust each other's app, but if DeSo explodes like we want it to, there could be some poor actors in the future who will take advantage of people's trust.

**Reply by iPaulPro** [reply]
Heads up that you can use [iPaulPro/overridentity](https://github.com/iPaulPro/overridentity) in the meantime.

**Reply by kennyjacobson** [reply]
Thanks!
